### PR TITLE
Add step for including a file into repodata

### PR DESCRIPTION
**Pull request by janblazek**
This patch adds a step for including a file into repodata.

It has following syntax.

(given) a file {} with type {} added into repository {}
"""
content
"""
use content if the provided file(path) doesn't exist

### CODE REVIEW

**Comment by ignatenkobrain** [comment]
seems like you need to rebase...

File: dnf-docker-test/features/steps/repo_steps.py [hunk 1]
```diff
@@ -707,3 +707,26 @@ def step_a_repo_file_of_repository_modified_with(ctx, repository):
...
+
+    if not os.path.isfile(filepath):
+        ctx.assertion.assertIsNotNone(ctx.text, "Multiline text is not provided")
+        tmpdir = tempfile.mkdtemp()
+        filepath = os.path.join(tmpdir, os.path.basename(filepath))
+        with open(filepath, 'w') as fw:
+            fw.write(ctx.text)
+
+    file_utils.set_dir_content_ownership(ctx, repodir, 'root')   # change file ownership to root so we can change it
+    cmd = "%s --mdtype=%s %s %s" % (modifyrepo, mdtype, filepath, os.path.join(repodir, "repodata"))
```
Comment (ignatenkobrain) [inline dnf-docker-test/features/steps/repo_steps.py]: please use `str.format()` instead

**Comment by ignatenkobrain** [comment]
Also please rebase

**Comment by kkaarreell** [comment]
Honzo, 
could you please also update the doc string with an example? Even something trivial would better than to have it empty.

**Comment by ignatenkobrain** [comment]
And also... rebase! ;)

**Comment by janblazek** [comment]
Sorry, I have created new pull request. https://github.com/rpm-software-management/ci-dnf-stack/pull/314

**Pull request closed without merging**